bcdagorm/models.go

```diff
@@ -0,0 +1,61 @@
+package bcdagorm
+
+import (
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	"github.com/pborman/uuid"
+
+	"github.com/CMSgov/bcda-app/bcda/database"
+)
+
+func Initialize() *gorm.DB {
+	db := database.GetGORMDbConnection()
+
+	defer db.Close()
+
+	// Migrate the schema
+	// Add your new models here
+	db.AutoMigrate(
```

Comment:
I'm a little worried about running migrations automatically when the app starts. Are these always safe to run? What happens when there are multiple instances of the app started across many servers?

Thinking it'd be nice to have this wrapped up in a `bcda migrate` command or similar to have control over the timing of migrations. What do you think?

I'm also wondering if there's a way to have gorm output the SQL it plans to apply for migrations. Maybe a [`bcda sqlmigrate`](https://docs.djangoproject.com/en/2.1/ref/django-admin/#django-admin-sqlmigrate) command?

Reply:
I think a sql migrate command makes sense.  I will change it.  It needs to be added to the jenkins deploy though.

Reply:
Added this command.